### ISSUE TITLE
Certificate issue behind company proxy

### ISSUE THREAD

**Issue by MaartenPetiet** [fenneclab/hugo-bin#66]
When I run npm install hugo-bin behind our company proxy it fails with following error:

```
> hugo-bin@0.16.0 postinstall D:\SOURCES\Netlify.VDW\node_modules\hugo-bin
> del-cli vendor && node lib/install

x unable to get local issuer certificate
x Hugo binary installation failed
```

We can see that the module is trying to download the following file: https://github.com/spf13/hugo/releases/download/v0.36.1/hugo_0.36.1_Windows-64bit.zip

When I try to download this file via my browser, it succeeds. But hugo-bin seems to use the module bin-wrapper which uses the module download which fails.

Even after setting strict-ssl to false (npm config set strict-ssl false) the issue continues.

I'm running Windows 2012 R2 and tried it with nodejs 8.11.3 and 10.8.0.

**Comment by satoshun00** [comment]
@MaartenPetiet 
Do you have same issue when run the [`download`](https://www.npmjs.com/package/download) module like following?

```js
const download = require('download');
 
download('https://github.com/spf13/hugo/releases/download/v0.36.1/hugo_0.36.1_Windows-64bit.zip', 'dist').then(() => {
    console.log('done!');
});

```

**Comment by MaartenPetiet** [comment]
@satoshun00 yes same issue.

**Comment by XhmikosR** [comment]
This should be fixed after #120
